docs/reference/mapping.asciidoc

```diff
@@ -123,44 +123,158 @@ You can create field mappings when you
 fields to an existing index with the <<indices-put-mapping,PUT mapping API>>.
 
 [float]
-== Updating existing field mappings
```

Comment:
Relocated to https://github.com/elastic/elasticsearch/pull/45520/files#diff-2443d9093ffa88ea9b13d16426fde216R179